tests/integration/FtpClientTest.php

```diff
@@ -131,7 +131,7 @@ public function testAsyncDownload() : void
 
         $client->createFile(self::$testFile, 'some content');
 
-        $this->assertTrue($client->asyncDownload(self::$testFile, $localFile, function () {
+        $this->assertTrue($client->asyncDownload(self::$testFile, $localFile, function ($speed, $percentage, $transferred, $seconds) {
```

Comment:
Since `PHP-8.0`, the `call_user_func_array` should have the named parameters.

Otherwise it will get the fatal `unknown parameter` error.